pre_commit/commands/run.py

```diff
@@ -254,11 +254,13 @@ def run(runner, store, args, environ=os.environ):
         repo_hooks = []
         for repo in repositories(runner.config, store):
             for _, hook in repo.hooks:
-                if (
-                    (not args.hook or hook['id'] == args.hook) and
-                    not hook['stages'] or args.hook_stage in hook['stages']
```

Comment:
I might be wrong but I think  adding parens around this second condition is sufficient (which would be a simpler fix). Let me know if that works :)